setup.py

```diff
@@ -122,7 +122,7 @@ def run(self):
         'importlib',
         'python-magic',
         'python-docx',
-        'ftfy',
+        'ftfy==4.4.1',  # the highest version that supports Python 2
```

Comment:
Why pin to a specific version instead of `ftfy<5`? (Which is recommended in the README of `ftfy`.) That way, we still get minor/patch releases in the 4.x.y series if they're published.